sdk/formrecognizer/ai-form-recognizer/tests.yml

```diff
@@ -20,3 +20,17 @@ stages:
         AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
         AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+      CloudConfig:
+        Public:
+          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+          Location: "canadacentral"
+        Canary:
+          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+          Location: "centraluseuap"
+        UsGov:
+          SubscriptionConfiguration: $(sub-config-gov-test-resources-test)
```

Comment:
Get rid of the `-test` suffix here. This was a testing keyvault and you are reminding me I need to go delete it :)

Reply:
Same for @maririos in the [NET repo](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/tests.yml#L16).